pkg/k8sutil/k8sutil.go

```diff
@@ -25,6 +28,22 @@ var (
 	ErrRunLocal     = fmt.Errorf("operator run mode forced to local")
 )
 
+// MustHaveKubeClient creates a new kubeclient that interacts with the Kube api with the service account secrets
+func KubeClient(s *runtime.Scheme) (client.Client, error) {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		panic(`MustHaveContainerClient: unable to obtain cluster config` + err.Error())
```

Comment:
nit: error message has wrong function name

Also, is panicking the right thing to do here on any error?